README.md

```diff
@@ -0,0 +1,95 @@
+# Online-books-store ![](media/logo.jpg)
+## Intro 
+This is the "Online Books Store" application based on Spring Boot, which allows you to manage a catalog of books and 
+handle user orders. The application includes the following features:
+
+- Adding and editing books to the catalog.
+- Managing user orders.
+- User authentication and authorization.
+- API documentation.
+- Containerization.
+
+## Technologies and Tools Used
+
+This project is built using a variety of technologies and tools to ensure a robust and efficient online book store. Here's an overview of the key components:
+
+- **Spring Boot**: This project is based on the Spring Boot framework, which simplifies the setup and development of Java applications.
+
+- **Spring Security**: Spring Security is used to handle authentication and authorization, ensuring that only authorized users can access certain endpoints.
+
+- **Spring Data JPA**: Spring Data JPA simplifies database access and management, allowing seamless interaction with the database using Java objects.
+
+- **Swagger**: Swagger is integrated into the project to provide interactive API documentation. You can explore and test the API endpoints using the Swagger UI.
+
+- **Docker**: Docker is used for containerization, making it easy to deploy and manage the application in various environments.
+
+- **Database**: The project uses a relational database MySQL to store book and user data. The choice of database can be configured to fit your requirements.
+
+- **Maven**: Maven is used for project management and dependency resolution. It simplifies the build and package process.
+
+
+This project leverages these technologies and tools to provide a secure, efficient, and user-friendly online book store experience. Whether you're a developer or user, these technologies work together to make the online book store a seamless and enjoyable platform.
+## How to use
+To start this project, you'll need to follow these general steps. Keep in mind that the specific instructions may vary based on the project's structure and your environment. Here's a high-level overview of how you can start this Spring Boot-based online bookstore project:
+
+- **Prerequisites**:
+
+Ensure you have Java Development Kit (JDK) installed on your system.
+Make sure you have Docker installed if you plan to run the application in a container.
+- **Clone the Repository**:
+
+Clone the project's repository from a version control system like Git.
+- **Database Configuration**:
+
+Check the project's configuration files (usually in the src/main/resources directory) to configure your database connection settings. Ensure you have a MySQL database set up and that you provide the correct credentials.

```

Comment:
Provide an option to run your app without manually installing MySQL and configuring a database. Provide a guide how to do all of that using Docker and docker-compose file.